source/eitri/eitri.py

```diff
@@ -1,4 +1,5 @@
-#!python
```

Comment:
Are you sure about removing this ? 
I'm afraid that removing this will break the binary packaging/integration.

Reply:
For what I tested the shebang doesn't work.
I have to run it with "python ./eitri" or "python2 ./eitri", I re-add an other shebang in #2852